GeeksCoreLibrary/Modules/Templates/Services/TemplatesService.cs

```diff
@@ -1163,6 +1166,29 @@ public async Task<JArray> GetJsonResponseFromQueryAsync(QueryTemplate queryTempl
             return result;
         }
 
+        /// <inheritdoc />
+        public async Task<JArray> GetJsonResponseFromRoutineAsync(RoutineTemplate routineTemplate, string encryptionKey = null, bool skipNullValues = false, bool allowValueDecryption = false)
+        {
+            var routineName = $"WISER_{routineTemplate.Name}";
+            // First get all parameters of the routine.
+            var query = @"SELECT PARAMETER_NAME
+FROM information_schema.parameters 
+WHERE SPECIFIC_NAME = ?name
+AND PARAMETER_MODE = 'IN'
```

Comment:
Is er een reden waarom je hier alleen `IN` parameters opvraagt, en niet `OUT` en `INOUT` parameters? Tenzij alle parameters worden opgegeven kan een procedure niet uitgevoerd worden. Voor functions maakt het niet uit, want die hebben alleen `IN` parameters.

Reply:
Omdat ik het verkeerd heb begrepen blijkbaar. Ik dacht dat `OUT` stond voor de return type van een function, die heb ik niet nodig. Ik had er niet aan gedacht dat je ook `OUT` en `INOUT` parameters hebt. Ik zal dat aanpassen.

Reply:
Ik heb dit nu aangepast.